### PR TITLE
fix description for the IOPS panels

### PR DESCRIPTION
**Pull request by cloudbehl**
tendrl-bug-id: Tendrl/monitoring-integration#497
bugzilla: 1587804
Signed-off-by: Ankush Behl <cloudbehl@gmail.com>

### CODE REVIEW

**Comment by cloudbehl** [comment]
@Tendrl/admins Don't merge the PR as it needs a round of testing!

File: etc/tendrl/monitoring-integration/grafana/dashboards/tendrl-gluster-bricks.json [hunk 1]
```diff
@@ -562,7 +562,7 @@
                 "dashLength": 10,
                 "dashes": false,
                 "datasource": "",
-                "description": "The IOPS panel displays the sum of IOPS (read + write) for a given brick path.",
+                "description": "The IOPS panel displays aggregated brick level read and write operations collected using “gluster volume profile info” over a period of time. The IOPS value shown only reflects metrics from volumes that have volume profiling enabled. No data will be shown if volume profiling is not enabled on at least 1 volume.",
```
Comment (nthomas-redhat) [inline etc/tendrl/monitoring-integration/grafana/dashboards/tendrl-gluster-bricks.json]: Why this update required at brick level?

File: etc/tendrl/monitoring-integration/grafana/dashboards/tendrl-gluster-hosts.json [hunk 2]
```diff
@@ -620,15 +620,15 @@
                 "dashLength": 10,
                 "dashes": false,
                 "datasource": "",
-                "description": "The IOPS panel displays IOPS for a given host over a period of time.  IOPS is based on the aggregated brick level read and write operations.",
+                "description": "The IOPS panel displays aggregated brick level read and write operations collected using “gluster volume profile info” over a period of time. The IOPS value shown only reflects metrics from volumes that have volume profiling enabled. No data will be shown if volume profiling is not enabled on at least 1 volume.",
```
Comment (nthomas-redhat) [inline etc/tendrl/monitoring-integration/grafana/dashboards/tendrl-gluster-hosts.json]: Same as above

File: etc/tendrl/monitoring-integration/grafana/dashboards/tendrl-gluster-volumes.json [hunk 3]
```diff
@@ -1055,7 +1055,7 @@
                 "dashLength": 10,
                 "dashes": false,
                 "datasource": "",
-                "description": "The IOPS panel displays volume IOPS based on brick-level read and write operations fetched using “gluster volume profile\".",
+                "description": "The IOPS panel displays aggregated brick level read and write operations collected using “gluster volume profile info” over a period of time. The IOPS value shown only reflects metrics from volumes that have volume profiling enabled. No data will be shown if volume profiling is not enabled on at least 1 volume.",
```
Comment (nthomas-redhat) [inline etc/tendrl/monitoring-integration/grafana/dashboards/tendrl-gluster-volumes.json]: Same here as well

File: etc/tendrl/monitoring-integration/grafana/dashboards/tendrl-gluster-bricks.json (hunk 1 above)
Comment (cloudbehl) [reply etc/tendrl/monitoring-integration/grafana/dashboards/tendrl-gluster-bricks.json]: As asked by @mbukatov on bug :https://bugzilla.redhat.com/show_bug.cgi?id=1587804#c16, these changes are made!

File: etc/tendrl/monitoring-integration/grafana/dashboards/tendrl-gluster-hosts.json (hunk 2 above)
Comment (cloudbehl) [reply etc/tendrl/monitoring-integration/grafana/dashboards/tendrl-gluster-hosts.json]: https://github.com/Tendrl/monitoring-integration/pull/498#discussion_r197356500

File: etc/tendrl/monitoring-integration/grafana/dashboards/tendrl-gluster-volumes.json (hunk 3 above)
Comment (cloudbehl) [reply etc/tendrl/monitoring-integration/grafana/dashboards/tendrl-gluster-volumes.json]: https://github.com/Tendrl/monitoring-integration/pull/498#discussion_r197356500

File: etc/tendrl/monitoring-integration/grafana/dashboards/tendrl-gluster-bricks.json (hunk 1 above)
Comment (nthomas-redhat) [reply etc/tendrl/monitoring-integration/grafana/dashboards/tendrl-gluster-bricks.json]: Probably need to use :
"The IOPS panel displays aggregated brick level read and write operations
collected using “gluster volume profile info” over a period of time. The IOPS
value shown only reflects metrics from volumes that have volume profiling
enabled. No data will be shown if the brick is not part of a volume for which the volume profiling is enabled."

File: etc/tendrl/monitoring-integration/grafana/dashboards/tendrl-gluster-volumes.json (hunk 3 above)
Comment (nthomas-redhat) [reply etc/tendrl/monitoring-integration/grafana/dashboards/tendrl-gluster-volumes.json]: Probably need to use :
"The IOPS panel displays aggregated brick level read and write operations
collected using “gluster volume profile info” over a period of time.  The IOPS
value shown only reflects metrics from volumes that have volume profiling
enabled.  No data will be shown if at-least one brick is not part of a volume for which the volume profiling is enabled."

**Pull request merged**